drgn/helpers/linux/bitops.py

```diff
@@ -20,15 +20,23 @@
 )
 
 
-def for_each_set_bit(bitmap: Object, size: IntegerLike) -> Iterator[int]:
+def for_each_set_bit(
+    bitmap: Object, size: Optional[IntegerLike] = None
+) -> Iterator[int]:
     """
     Iterate over all set (one) bits in a bitmap.
 
     :param bitmap: pointer to, or array of, unsigned integer type,
         EG: ``unsigned long *``
-    :param size: Size of *bitmap* in bits.
+    :param size: Size of *bitmap* in bits. When *bitmap* is a sized array type
+        (EG: ``unsigned long[2]``), this can be omitted.
```

Comment:
It can be a bug to assume that the whole array is being used. See ee0b8efc303329bdb52f25c9acfef108f5c40ddc.

I still like this change for 99% of cases, so to hopefully avoid the buggy case, I'd change this wording to say that it defaults to the size of the array in bits, rather than saying the size can be omitted.